### DISCUSSION TITLE
Why are the preview in the MD file and the rendering results in the browser inconsistent?

### DISCUSSION THREAD

**Discussion by krotia** [academicpages/academicpages.github.io#2232] [Q&A]
Why are the preview in the MD file and the rendering results in the browser inconsistent? Moreover, why are the results inconsistent even when I use nearly identical repeated content blocks with the same format in the MD file?
This is the result rendered in the browser, and this is the corresponding block in the MD file.
![微信图片_20240704114428](https://github.com/academicpages/academicpages.github.io/assets/50139631/99bb304c-4809-4004-b253-d5a9c5df9fb6)
and this is the corresponding block in the MD file.
![微信图片_20240704114434](https://github.com/academicpages/academicpages.github.io/assets/50139631/112d8672-d0c0-402b-9d81-8cfe6fa783eb)

**Comment by krotia** [comment]
By the way, I have pre-processed these images to ensure they are all the same size.

**Comment by rjzupkoii** [comment]
This is something that needs a bit more documentation for the template, but depending upon where you are previewing the Markdown rendering, the results are going to be a bit different. This is also part of the reason why not all Markdown emojis (see #534) that people are used to supporting are available.

**Reply by krotia** [reply]
Thanks for the reply.
Actually, what puzzles me the most is why images of the same size and with the same formatting are rendered differently. This may not be an error with the template; rather, there might be rendering issues that I have not discovered or understood.

**Reply by rjzupkoii** [reply]
@krotia Well the code that you posted in the first post is actually HTML instead of Markdown - is that the HTML that was output after processing the Markdown or is that what you entered yourself? Usually when images are different sizes but the the HTML is the same it's something about the images themselves, so I typically start by checking their properties using developer mode in the web browser.
